pkg/universe.dagger.io/ubuntu/test/test.bats

```diff
@@ -0,0 +1,8 @@
+setup() {
+    load '../../bats_helpers'
```

Comment:
`../../../../bats_helpers` since your package will be in `universe.dagger.io/x/<your email>`